zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -196,7 +205,15 @@ public static List<Host> getTheTrainHosts() {
     }
 
     public static List<Host> getWebsiteHosts() {
-        return Arrays.asList(StringUtils.split(defaultIfBlank(getValue("website_url"), DEFAULT_WEBSITE_URL), ","))
+        String defaultLowercaseUrl = "website_url";
+        String defaultUrl = DEFAULT_WEBSITE_URL;
+
+        if (isCacheAPIEnabled()){
```

Comment:
Rather than overwriting the variables, I think it would be more readable to make this an if/else, or to use the ternary operator:
```java
String defaultUrl = isCacheAPIEnabled() ? DEFAULT_CACHE_API_URL : DEFAULT_WEBSITE_URL;
```

Reply:
we chose to use ternary operator